### ISSUE TITLE
[Image]: VO gets stuck on image and cannot navigate further – React Safari Mobile

### ISSUE THREAD

**Issue by oliviaflory** [carbon-design-system/carbon-for-ibm-dotcom#8536]
### Description

VO is getting stuck on the image, repeating “Image, image alt text, Image, image alt text...” as I try to navigate past it. This seems to happen when I try to swipe too quickly, but I have noticed it on multiple components: Callout with media, Content block media, Content block segmented, possibly image cards.

Workaround: refresh the page and allow VO to read out the full image context before swiping.

https://user-images.githubusercontent.com/45692622/158699519-27bf7576-cbf1-4325-96c4-f159b7b44f54.MOV

### Component(s) impacted

Image with caption
Image

### Browser

Safari

### Carbon for IBM.com version

v1.32.0 rc0

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

c4IBM.com

### Package

@carbon/ibmdotcom-react

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/react/?path=/story/components-content-block-segmented--default

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by IgnacioBecerra** [comment]
@oliviaflory I was talking with @annawen1 and @ariellalgilmore about this issue, and it seems that this might be a VoiceOver related bug with the new Safari look since it seems to happen randomly. Upon load, the image is directly behind the Safari menu, possibly making it "get stuck". Ari pointed out that if enabling the setting to minimize the Safari bar, VO never gets stuck.

Default view (gets stuck every once in a while):
![image](https://user-images.githubusercontent.com/24970122/160015876-ba128bdc-1df1-453e-ab0a-d0cdcd269a4f.png)

vs

Minimized (never gets stuck)
![image](https://user-images.githubusercontent.com/24970122/160016566-c7e168de-62e3-42c3-9fcf-434f87d37a4c.png)

Wondering what are your thoughts on this, since the Image component doesn't seem to be the cause of these VO issues.

**Comment by oliviaflory** [comment]
@IgnacioBecerra wow that is so interesting. Sounds like if it's a bug with Safari we don't do anything on our end? cc: @kennylam what are your thoughts?

**Comment by kennylam** [comment]
@oliviaflory If this is the case I suggest we don't include this in an `rc` and continue to test. Seems like it is difficult to reproduce and might not affect all users? We can continue to test this and push to next sprint.

**Comment by IgnacioBecerra** [comment]
Still can't replicate this issue, will be closing this.
